jerry-core/parser/js/js-scanner.c

```diff
@@ -202,6 +201,12 @@ scanner_scan_primary_expression (parser_context_t *context_p, /**< context */
       break;
     }
 #if ENABLED (JERRY_ESNEXT)
+    case LEXER_KEYW_SUPER:
```

Comment:
Put this before LEXER_KEYW_THIS and use fallthrough.

Reply:
There is already a fallthru for LEXER_KEYW_THIS.